examples/xed-examples-util.c

```diff
@@ -219,7 +219,7 @@ xed_int64_t xed_atoi_general(char* buf, int mul) {
         {
             p++;
         }
-        while(*p && isdigit(*p))
+        while(*p && isdigit((unsigned char)*p))
```

Comment:
ditto. would this be better as (int)? That seems like it matches the man pages on win/lin/mac and netbsd